### PR TITLE
Add requirements ruby version

### PR DESCRIPTION
**Pull request by xluffy**

### CODE REVIEW

**Comment by xluffy** [comment]
For a job `Test / integration-tests`, I run in local PC, It works well. Not sure why.

```
bundle exec rspec ./spec/meilisearch/index/updates_spec.rb

MeiliSearch::Index - Updates
  gets an empty array when nothing happened before
  gets update status after adding documents
  gets all the update status
  waits for pending update with default values
  waits for pending update with default values after several updates
  waits for pending update with custom timeout_in_ms and raises MeiliSearchTimeoutError
  waits for pending update with custom interval_in_ms and raises Timeout::Error

Finished in 0.72082 seconds (files took 0.59354 seconds to load)
7 examples, 0 failures

Coverage report generated for RSpec to /Users/xluffy/code/meilisearch-ruby/coverage. 68 / 80 LOC (85.0%) covered.```

File: README.md [hunk 1]
```diff
@@ -41,6 +41,11 @@ See our [Documentation](https://docs.meilisearch.com/guides/introduction/quick_s
 
 ## 🔧 Installation
 
+### Requirements <!-- omit in toc -->
+
+- Ruby version 2.6.0 or later
+- httparty >= 0.17.1
+
```
Comment (curquiza) [inline README.md:48]: I would rather add a simple sentence.

"This package requires Ruby version 2.6.0 or later"

File: README.md [hunk 2]
```diff
@@ -53,6 +58,25 @@ source 'https://rubygems.org'
...
+  Version: 0.15.0
+  File: meilisearch-0.15.0.gem
+
+> gem install meilisearch-0.15.0.gem
+  Successfully installed meilisearch-0.15.0
+  Parsing documentation for meilisearch-0.15.0
+  Done installing documentation for meilisearch after 0 seconds
+  1 gem installed
+```
+
```
Comment (curquiza) [inline README.md:79]: I'm not sure this is useful in the README, this makes it too complex, and the README needs to stay simple 🙂

**Review by curquiza — CHANGES_REQUESTED** [review]
Thanks for adding this!! And sorry for the delay!

I request some changes in the README, otherwise, it looks ok! Thanks a lot!

**Comment by xluffy** [comment]
@curquiza Hey, I updated README now. Can you check again?

**Review by curquiza — APPROVED** [review]
Thanks a lot @xluffy!
And sorry again for the delay

bors merge

**Comment by curquiza** [comment]
bors merge

**Pull request merged**